ckanext/ckanext-apply_permissions_for_service/ckanext/apply_permissions_for_service/views.py

```diff
@@ -128,6 +132,15 @@ def view(application_id):
         toolkit.abort(403, toolkit._(u'Not authorized to see this page'))
 
 
+def preview(subsystem_id):
+    try:
+        context = {u'user': toolkit.g.user, u'auth_user_obj': toolkit.g.userobj}
+        toolkit.check_access('service_permission_application_create', context, {})
```

Comment:
Maybe you should use here some other permission instead of the one used for creation, e.g. "service_permission_settings_show"?

Reply:
Good catch, I changed it